universal-login-example/src/services/Threebox.js

```diff
@@ -0,0 +1,22 @@
+import Box from '3box';
+
+class ThreeBox {
+  constructor(provider) {
+    console.log(Box);
```

Comment:
There should be no console.log in production code

Reply:
@marekkirejczyk, was intended for debugging purposes. But, thanks for the heads up.
The code there right now is for reviewing purposes (as I was confused about the best method of implementation at the time).